source/_posts/marionette-v3.md

```diff
@@ -0,0 +1,243 @@
+title: Marionette.js v3. Getting started with new version 
+date: 2016-08-10 21:19:00 
+tags: 
+- backbone
+- marionette
+- learning
+- v3
+
+---
+
+New version of `Marionette.js` framework was released.
+
+Welcome to the party
```

Comment:
Maybe just:
> Welcome to the party.
> We will go through all new changes which v3 brought us.

I agree it's a cool framework, but I wouldn't want to get other framework Zealots responding to the opening line